KarateTraining/src/test/java/signuptask/signup.feature

```diff
@@ -0,0 +1,101 @@
+Feature: Testing the react herokuapp for Sign up users
+
+  Background:
+    * url baseUrl
+
+  Scenario: Sign up a new user in the app
+
+    * def randomString =
+    """
+    function(nbCharacters) {
+       var text = '';
+       var possible = 'abcdefghijklmnñopqrstuvwxyz0123456789';
+       for(var i = 0; i < nbCharacters; i++) {
+         text += possible.charAt(Math.floor(Math.random() * possible.length));
+       }
+       return text
+    }
+    """
```

Comment:
I like this! move this code to a single js file where you can reuse that if you want.